examples/com/amazonaws/examples/AsymmetricEncryptedItem.java

```diff
@@ -0,0 +1,95 @@
+package com.amazonaws.examples;
+
+import java.nio.ByteBuffer;
+import java.security.GeneralSecurityException;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.encryption.DynamoDBEncryptor;
+import com.amazonaws.services.dynamodbv2.datamodeling.encryption.EncryptionContext;
+import com.amazonaws.services.dynamodbv2.datamodeling.encryption.EncryptionFlags;
+import com.amazonaws.services.dynamodbv2.datamodeling.encryption.providers.WrappedMaterialsProvider;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+
+/**
+ * Example showing use of RSA keys for encryption and signing.
+ * For ease of the example, we create new random ones every time.
+ */
+public class AsymmetricEncryptedItem {
+ 
+  public static void main(String[] args) throws GeneralSecurityException {
+    final String tableName = args[0];
+    final KeyPairGenerator keyGen = KeyPairGenerator.getInstance("RSA");
+    keyGen.initialize(2048);
+    // You should never use the same key for encryption and signing
+    final KeyPair encryptionKeys = keyGen.generateKeyPair();
+    final KeyPair signingKeys = keyGen.generateKeyPair();
+    
+    encryptRecord(tableName, encryptionKeys, signingKeys);
+  }
+
+  private static void encryptRecord(String tableName, KeyPair encryptionKeys, KeyPair signingKeys) throws GeneralSecurityException {
```

Comment:
To avoid confusion, can we call the wrapping keypair `wrappingKeys` instead of `encryptionKeys`?

Reply:
Renamed here and in `SymmetricEncryptedItem`.